Projects/Applications/FreeRTOS/FreeRTOS_LoRaWAN/Core/App/Src/receive.c

```diff
@@ -9,4 +11,40 @@ void Process_Rx_Data(const LmHandlerAppData_t *const app_data, const LmHandlerRx
     	APP_LOG(TS_OFF, VLEVEL_M, "%02X ", app_data->Buffer[i]);
     }
     APP_LOG(TS_OFF, VLEVEL_M, "\r\n");
+    Interpret_Message(app_data->Buffer, app_data->BufferSize);
+}
+
+void Interpret_Message(uint8_t *Buffer, uint8_t BufferSize)
+{
+    if (BufferSize < 2) {
+        APP_LOG(TS_OFF, VLEVEL_M, "Message too short to interpret\n");
+        return;
+    }
```

Comment:
hier ook een lege regel zou mooi zijn
```suggestion
    }
    
```